code/modules/games/cards.dm

```diff
@@ -118,15 +130,17 @@
 
 	if(!H)
 		H = new(get_turf(src))
+		H.dir = NORTH
 		user.put_in_hands(H)
 
 	var/datum/playingcard/P = cards[1]
 	H.cards += P
 	cards -= P
 	H.parentdeck = src
+	H.update_values()
 	H.update_icon()
 	user.visible_message("<span class='notice'>[user] draws a card.</span>","<span class='notice'>You draws a card.</span>")
-	to_chat(user,"<span class='notice'>It's [P].</span>")
+	to_chat(user,"<span class='notice'>It's the [P].</span>")
```

Comment:
Double-check that this doesn't double up and turn into "the the queen of hearts"